keras/activations.py

```diff
@@ -34,6 +34,20 @@ def elu(x, alpha=1.0):
     return K.elu(x, alpha)
 
 
+def selu(x, alpha=1.6732632423543772848170429916717, scale=1.0507009873554804934193349852946):
```

Comment:
No point in providing those as arguments (we already have `elu` if you want custom values). They should be hard-coded.